common/source/docs/common-paramosd.rst

```diff
@@ -0,0 +1,134 @@
+.. _common-paramosd:
+
+========================
+OSD based parameter menu
+========================
+
+.. image:: ../../../images/paramscreen.jpg
+    :target: ../_images/paramscreen.jpg
+    :width: 450px
+
+This allows to set and adjust paramaters using the ardupilot maxchip-based onboard OSD and an RC transmitter's stick input. 
```

Comment:
parameter

Reply:
ArduPilot
Maxim chip based

Reply:
fixed. no need to mention the chipset at all.